### ISSUE TITLE
standalone attribute for `Section` and `Property` objects.

### ISSUE THREAD

**Issue by msuzen** [G-Node/python-odml#95]
Following discussion #93 

* Introduce `standalone` attribute for `Section` and `Property` objects. If `parent` is not explicitly set, default `standalone` could be `yes`.
* If defined as standalone, the object should be immutable and not attachable to another object. Use case could be a similar sections in multiple experiments could reoccur, so one could copy one of those in different experiment metadata.

**Comment by lzehl** [comment]
I disagree especially with the second point! Please cf. My comment of this in #93

**Comment by msuzen** [comment]
It was a high level suggestion, implementation detail/object can be different of course. But some sort of `immutable` structure/object hierarchy would be nice to have in case loading a re-usable chunk. c.f. my comments #93.

**Comment by lzehl** [comment]
@msuzen: ahh :) now I get it. This goes into the direction of my request for real template structures/objects for odml files! This would be awesome, but as you said a high level suggestion.

**Comment by msuzen** [comment]
@lzehl  Cool! I think your labeling of the issue sounds more appropriate, it is basically a functionality that facilitates templates in API level. But please note that, what I was trying to explain is not exactly a set of XML templates, more of a workflow-like functionality within odML. 

In summary, out of our discussions, anything am I missing?, a wish/feature list:

* Specify if the object that is outputted into file is a chunk, `write_section` or similar method with a mechanism for this, for example, by introducing `standalone` attribute, as  mentioned in #93 as well.
* Read `standalone` odML file and make it an immutable object.
* When appending a `standalone` odML chunk, copy the object and to prevent overwriting re-occuring chunk.
* Adding a list of dynamic properties, i.e., variables that should be specified when appending a `standalone` object.

**Comment by msuzen** [comment]
c.f. #45 too
